template-parts/single.php

```diff
@@ -9,7 +9,7 @@
 ?>
 <?php while ( have_posts() ) : the_post(); ?>
 
-<main id="main" class="site-main <?php post_class(); ?>" role="main">
+<main id="main" <?php post_class('site-main'); ?> role="main">
```

Comment:
Please wrap the argument with spaces. 
(https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/develop/WordPress-Core/ruleset.xml#L217)

Reply:
Done ! (not totally sure cause i've rarely used github)